### DISCUSSION TITLE
Reverse Proxy I can't configure it

### DISCUSSION THREAD

**Discussion by VlaK0r** [Athou/commafeed#1120] [Q&A]
Hello!
Thank you for your program.

After installing the program, it works www.domen.com:8082
I want it to be like this  www.domen.com/test
To do this, I make a configuration file in apache:

`<VirtualHost *:80>`
`ServerName domen.com`
`ServerAlias www.domen.com`
`ProxyPreserveHost On`
   ` <IfModule mod_proxy.c>`
      `  ProxyRequests off`
      `  ProxyVia on`
     `   ProxyPass /test http://localhost:8082/`
       ` ProxyPassReverse /test http://localhost:8082/`
          `  <Proxy *>`
           ` Order deny,allow`
        `    Allow from all`
        `  </Proxy>`
    `  </IfModule>`
`</VirtualHost>`

Go to the address    www.domen.com/test
The page starts to load, but then nothing happens, just a white page in the browser.

If you change the file

`<VirtualHost *:80>`
  ` ServerName domen.com`
    `ServerAlias www.domen.com`
` ProxyPreserveHost On`
    `<IfModule mod_proxy.c>`
      `  ProxyRequests off`
      `  ProxyVia on`
       ` ProxyPass / http://localhost:8082/`
      `  ProxyPassReverse / http://localhost:8082/`
        `<Proxy *>`
        `    Order deny,allow`
        `    Allow from all`
       ` </Proxy>`
   ` </IfModule>`
`</VirtualHost>`

When going to the address www.domen.com
Everything works fine.

What is my mistake, please help.

**Comment by Athou** [comment]
Haven't tried that in a while, but this should still work: https://github.com/Athou/commafeed/issues/767

https://www.dropwizard.io/en/latest/manual/configuration.html#simple

**Reply by VlaK0r** [reply]
Is this for Apache?

**Reply by Athou** [reply]
No, it's for your CommaFeed config.yml file. CommaFeed is using the dropwizard framework for the backend.

**Reply by VlaK0r** [reply]
I'll try it now.

**Reply by VlaK0r** [reply]
It didn't help
![image](https://github.com/Athou/commafeed/assets/77666575/f77462ba-3ed8-47be-91bb-b62cb7478e5f)

**Reply by VlaK0r** [reply]
It doesn't go any further than that

**Reply by VlaK0r** [reply]
The most interesting thing is that this file works with rssbridge.
I have it installed through docker and it opens by default http://rssbridge:3000

**Reply by VlaK0r** [reply]
Maybe I did something wrong?
![image](https://github.com/Athou/commafeed/assets/77666575/83b4943f-518e-4241-8c26-799bcac23f3b)
